lib/srv/server/ec2_watcher.go

```diff
@@ -30,9 +30,9 @@ import (
 	usageeventsv1 "github.com/gravitational/teleport/api/gen/proto/go/usageevents/v1"
 	"github.com/gravitational/teleport/api/types"
 	"github.com/gravitational/teleport/lib/cloud"
+	taws "github.com/gravitational/teleport/lib/cloud/aws"
```

Comment:
Can we re-use one of the already existing alias for this import?
`libcloudaws` or `awslib`